documentation/staging/content/userguide/managing-operators/conversion-webhook.md

```diff
@@ -0,0 +1,157 @@
+
+| Helm command with the operator Helm chart | Helm chart custom values | Behavior |
+| --- | --- | --- |
+| Helm install | Default | Operator and conversion webhook installed. |
+| Helm install | Default and existing `weblogic-operator-webhook` deployment | Operator installed. |
+| Helm install | `webhookOnly=true` | Conversion webhook installed. |
+| Helm install | `preserveWebhook=true` | Operator and webhook installed. |
+| Helm uninstall | | Operator and webhook deployment uninstalled. |
+| Helm uninstall with `preserveWebhook=true` set during `helm install` | | Operator deployment uninstalled (and webhook deployment preserved). |
+
+#### Uninstall the conversion webhook
+When you [uninstall the operator]({{<relref "/userguide/managing-operators/installation#uninstall-the-operator">}}) using the `helm uninstall` command, it removes the conversion webhook 
+associated with the release and its resources from the Kubernetes cluster. However, if you **only** installed the conversion webhook using `webhookOnly=true` custom value, then run the `helm uninstall` 
+command separately to remove the conversion webhook associated with the release and its resources.
+
+For example, assuming the Helm release name for the conversion webhook is `sample-weblogic-conversion-webhook`,
+and the conversion webhook namespace is `sample-weblogic-conversion-webhook-ns`:
+
+```text
+$ helm uninstall sample-weblogic-conversion-webhook -n sample-weblogic-conversion-webhook-ns
+```
+This command deletes the conversion webhook Deployment (`weblogic-operator-webhook`), and it also deletes the conversion resources such as service and Secrets.
+
+{{% notice warning %}}
+The `helm uninstall` command does not delete the conversion definition stored in the Domain CRD. However, the remaining conversion definition in the CRD will cause problems if the conversion webhook runtime is not available. When this happens, you will see a `connection refused` error when creating a Domain using WKO 3.x/V8 domain resource. To avoid this error, manually patch the Domain CRD to set the conversion strategy to `None` and remove the `Webhook` details using the below command. Alternatively, use the `preserveWebhook=true` custom value when installing the conversion webhook using the `helm install` command to prevent the removal of the conversion webhook. 
+{{% /notice %}}
+
+Use the below command to manually patch the Domain CRD to set the conversion strategy to `None` and remove the `Webhook` details:
+```
+ kubectl patch crd domains.weblogic.oracle --type=merge --patch '{"spec": {"conversion": {"strategy": "None", "webhook": null}}}'
+```
+
+When you run the `helm install` command again to install a new WebLogic Domain resource conversion webhook, the conversion webhook runtime updates the CRD conversion strategy to `Webhook`. It also updates the necessary webhook client configuration details.
+
+#### Troubleshooting the conversion webhook
```

Comment:
Suggest moving this material into 'troubleshooting' guide and instead adding a reference to the 'debugging' guide (which in turn points to 'troubleshooting').

Reply:
Moved to `troubleshooting` guide and added a reference to the `troubleshooting the conversion webhook` section in `troubleshooting` guide. I didn't add referece to the the `debugging` guide as the conversion webhook errors can happen before the Domain gets created.